docs/Tutorials/VisualizationWithPython.ipynb

```diff
@@ -0,0 +1,244 @@
+{
+ "cells": [
```

Comment:
So we already have a "Visualizing volume data" [tutorial](https://spectre-code.org/tutorial_visualization.html). This feels like a very natural extension of that tutorial, having two sections

- Visualizing with paraview GUI
- Visualizing with spectre python

Reply:
I added a section there with a link to the notebook (can't merge the tutorials because this is a notebook).